package.json

```diff
@@ -19,7 +19,9 @@
     "stylus": "~0.54.5",
     "uglify-js": "^2.7.5"
   },
-  "devDependencies": {},
+  "devDependencies": {
+    "jsonlint": "^1.6.2"
```

Comment:
> When using ^ I am sure you will never look back and check for new major versions.

I mean this is also an assumption in part. I don't see why can't just check for new versions ourselves. I feel like a contributor's (potential) experience is more important than our convenience. I see what you mean, not having the latest ruleset could also result in a bad experience for the contributor, but again, we can have the best of both worlds at the cost of some imo very minor convenience. :v: